q2_diversity/tests/test_alpha_rarefaction.py

```diff
@@ -145,11 +145,11 @@ def test_invalid_invocations(self):
 
             with self.assertRaisesRegex(ValueError, 'phylogeny was not'):
                 alpha_rarefaction(output_dir, t, max_depth=200,
-                                  metadata=md, metric='faith_pd')
+                                  metadata=md, metrics=set(['faith_pd']))
 
             with self.assertRaisesRegex(ValueError, 'Unknown metric: pole'):
                 alpha_rarefaction(output_dir, t, max_depth=200,
-                                  metadata=md, metric='pole-position')
+                                  metadata=md, metrics=set(['pole-position']))
```

Comment:
https://en.wikipedia.org/wiki/Pole_Position_(video_game)

Reply:
> 1x ZiLOG Z80 @ 3.072 MHz,

That power! Haha